assets/sass/patterns/organisms/content-aside.scss

```diff
@@ -124,72 +132,56 @@
     .content-aside__first-column {
       @media only screen and (min-width: #{get-rem-from-px($bkpt-site--medium)}em) {
         @include padding(48, "right");
+        width: 50%;
       }
 
       @media only screen and (min-width: #{get-rem-from-px($bkpt-site--extended-wide)}em) {
         @include padding(0, "right");
+        width: 100%;
       }
     }
   }
 
-  .js & .content-aside__definition-list {
-    display: grid;
+  .content-aside__definition-list {
+    @include margin(36, "bottom");
 
-    @media only all and (min-width: #{get-rem-from-px($bkpt-site--medium)}em) {
-      display: block;
+    @media only screen and (min-width: #{get-rem-from-px($bkpt-site--extended-wide)}em) {
+      @include margin(48, "bottom");
     }
+  }
+
+  .js & .content-aside__definition-list {
+    display: grid;
 
     & .definition-list dt,
     & .definition-list dd {
       &:nth-of-type(n+2) {
         display: none;
       }
-
-      @media only all and (min-width: #{get-rem-from-px($bkpt-site--medium)}em) {
-        &:nth-of-type(-n+3) {
-          display: block;
-        }
-        &:nth-of-type(n+5) {
-          display: none;
-        }
-      }
-      @media only all and (min-width: #{get-rem-from-px($bkpt-site--extended-wide)}em) {
-        &:nth-of-type(n+2) {
-          display: block;
-        }
-      }
     }
 
     .toggle {
-      @include font-size-and-vertical-height(12, 24);
-      @include padding(0 17 0 10);
-      align-self: start;
+      @include font-size-and-vertical-height(14, 24);
+      @include margin(18, "top");
```

Comment:
Can we put the margin top on the bottom of the dl?

Reply:
Discussed, we will keep the margin top because the toggle is optional and won't be visible when there is one element in the timeline.